docs/protocol.html

```diff
@@ -114,6 +114,32 @@
 
 <p>Currently all versions are baselined at 0, as we evolve these APIs we will indicate the format for each version individually.</p>
 
+<h5><a id="api_versions" href="#api_versions">Retrieving Supported API versions</a></h5>
+<p>In order for a client to successfully talk to a broker, it must use requests versions' supported by the broker. Clients may work against
+    multiple broker versions, however to do so the clients need to know what versions of various APIs does a broker supports. Clients may
+    use the supported API versions information to take appropriate actions, like, propagate unsupported api version error to application
+    or choose a version from many API versions supported by client. The following sequence maybe used by a client to obtain supported API
```

Comment:
choose a version from many API versions supported by client -> choose an API request/response version supported by both the client and broker